cli/config/credentials/default_store_linux.go

```diff
@@ -5,7 +5,8 @@ import (
 )
 
 func defaultCredentialsStore() string {
-	if pass.PassInitialized {
+	pass := pass.Pass{}
```

Comment:
nit:
Too much pass 😵 
And not fan of shadowing a package by a variable.

Reply:
yeah same, besides that, LGTM!